internal/testplanet/planet.go

```diff
@@ -350,6 +366,67 @@ func (planet *Planet) newStorageNodes(count int) ([]*storagenode.Peer, error) {
 	return xs, nil
 }
 
+// newBootstrap initializes the bootstrap node
+func (planet *Planet) newBootstrap() (peer *bootstrap.Peer, err error) {
+	defer func() {
+		planet.peers = append(planet.peers, peer)
+	}()
+
+	prefix := "bootstrap"
+	log := planet.log.Named(prefix)
+	dbDir := filepath.Join(planet.directory, prefix)
+
+	if err := os.MkdirAll(dbDir, 0700); err != nil {
+		return nil, err
+	}
+
+	identity, err := planet.NewIdentity()
+	if err != nil {
+		return nil, err
+	}
+
+	db, err := bootstrapdb.NewInMemory(dbDir)
+	if err != nil {
+		return nil, err
+	}
+
+	// TODO: err = db.CreateTables()
+	// if err != nil {
+	// 	return nil, err
+	// }
+
+	planet.databases = append(planet.databases, db)
+
+	config := bootstrap.Config{
+		Server: server.Config{
+			Address:            "127.0.0.1:0",
+			RevocationDBURL:    "bolt://" + filepath.Join(planet.directory, "revocation.db"),
```

Comment:
This should be `filepath.Join(dbdir, "revocation.db")`. If it's copy-paste, then the bug is elsewhere as well.

Reply:
Yes, it was a copy-paste. I will fix it in `newStorageNodes` too.